### PR TITLE
Revive NEWS file from GIT history

### PR DESCRIPTION
**Pull request by vshlapakov**
Revive NEWS file based on GIT history.

Review, please.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,9 +1,117 @@
 python-scrapinghub release notes
 ================================
 
+Version 1.9.0
+-------------
+(release on Nov 2nd, 2016)
```
Comment (chekunkov) [inline NEWS]: why not to include date in the header, seems to be common practice

https://github.com/scrapy/scrapy/blob/master/docs/news.rst
https://github.com/kennethreitz/requests/blob/master/HISTORY.rst

I like requests approach `<version> (<date>)` the most - short and clean

File: NEWS [hunk 2]
```diff
@@ -1,9 +1,117 @@
...
+
+- python-hubstorage merged into python-scrapinghub
+- all tests are improved and rewritten with py.test
+- hubstorage tests use vcrpy cassettes, work faster and don't require any external services to run
+
+python-hubstorage is going to be considered deprecated,
```
Comment (chekunkov) [inline NEWS]: can we add a link to python-hubstorage?

File: NEWS (hunk 1 above)
Comment (redapple) [inline NEWS]: Note that starting with a digit seems to cause trouble with generated link (for Sphinx at least)
e.g for 2.12.3 (2016-12-01), the link is
`http://docs.python-requests.org/en/master/community/updates/#id1`

I "fixed" it in Scrapy by prepending "Scrapy " (https://github.com/scrapy/scrapy/pull/2294)

Comment (chekunkov) [inline NEWS]: that's strange @redapple we'll change the headers later if starting digit will stand in our way :)

**Pull request merged**